lib/transforms/onnxextension_legalizer.cc

```diff
@@ -33,6 +33,29 @@
 
 namespace halo {
 
+template <typename T>
+static const T& GetAttributeValue(const Attribute& attr) {
+  return T();
+}
+
+template <>
+const std::vector<int>& GetAttributeValue(const Attribute& attr) {
+  return attr.GetValueAsIntegerList();
+}
+
+template <typename T>
+static const T& FindAttributeValue(const ExtensionInst* ext,
+                                   const std::string& name,
+                                   const T& default_val) {
+  T ret_val = default_val;
+  for (const auto& it : ext->GetAttributes()) {
+    if (it->GetName() == name) {
+      return GetAttributeValue<T>(*it);
+    }
+  }
+  return default_val;
+}
+
```

Comment:
there are same codes in caffeextension_legalizer.cc .can we refine it in transforms_util?

Reply:
OK